openshift_tools/monitoring/hawk_sender.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+# vim: expandtab:tabstop=4:shiftwidth=4
+"""
+Collect metrics and send metrics to Hawk.  The data
+being send to Hawk is done using REST API using the HawkClient
+module
+
+Examples:
+
+    from openshift_tools.monitoring.hawk_common import HawkConnection, HawkHeartbeat
+    from openshift_tools.monitoring.hawk_sender import HawkSender
+    HOSTNAME = 'use-tower1.ops.rhcloud.com'
```

Comment:
Not sure where you got this hostname, but please use something else. Something like `host.example.com`.